Makefile

```diff
@@ -6,7 +6,7 @@ image:
 	docker build --rm -t $(IMAGE_NAME) .
 
 test: image
-	docker run --tty --interactive --rm $(IMAGE_NAME) bundle exec rake
+	docker run -e PAGER=more --tty --interactive --rm $(IMAGE_NAME) bundle exec rake
```

Comment:
yeah, i'm not sure the best way to handle the `PAGER` issue. another option is to do `--env-file .env.test` and just make a `.env.test` file with `PAGER=more`.  i guess since there's only one env variable here, the way you have it is probably fine.